keyboards/handwired/d48/README.md

```diff
@@ -0,0 +1,87 @@
+# D48
+
+![Proton C based handwired 40% keyboard](https://i.imgur.com/2wCYuno.jpg)
+
+A Proton C based handwired 48 key keyboard with 2 rotary encoders, I2C OLED, WS2812 strip, buzzer & clock!
+
+- Keyboard Maintainer: Andrew Dunai
+- Hardware Supported: Proton C handwired
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/d48:default:dfu-util
```

Comment:
```suggestion
    make handwired/d48:default
```

Reply:
@fauxpark Wouldn't it be better to include `:dfu-util` as an extra emphasis that this is a Proton board which needs `dfu-util` programmer?

Reply:
This section is just for the make example. You can use `:flash` and it will run the appropriate tool based on the value of `BOOTLOADER`.